docs/e2e/requirements.md

```diff
@@ -0,0 +1,88 @@
+| `Environment Variable` | `AZURE_REGION` | The region for an existing cluster |
+| `Environment Variable` | `RESOURCEGROUP` | The resource group for an existing cluster |
+
+#### Fake RP environment requirements
+
+The following are required for running all e2e tests against the fake RP
+
+| Artifact Kind | Name | Notes |
+| --- | --- | --- |
+| `Ginkgo Focus` | `[Fake]` | Allows the test to configure its `OpenshiftManagedCluster` client against the fake RP |
+| `Environment Variable` | `AZURE_CLIENT_ID` | Required for authentication against the RP |
+| `Environment Variable` | `AZURE_CLIENT_SECRET` | Required for authentication against the RP |
+| `Environment Variable` | `AZURE_TENANT_ID` | Required for authentication against the RP |
+| `Environment Variable` | `AZURE_SUBSCRIPTION_ID` | The subscription id for an existing cluster |
+| `Environment Variable` | `RESOURCEGROUP` | The resource group for an existing cluster |
+| `File` | `_data/containerservice.yaml` | Allows the test to retrieve credentials as well as to decide if it is a `create` vs `update` |
+| `File` | `_data/manifest.yaml` | The external cluster manifest (required for key rotation and reentrant updates tests) |
+| `File` | `secrets/logging-int.cert` | Geneva logging client certificate |
+| `File` | `secrets/logging-int.key` | Geneva logging client key |
+| `File` | `secrets/metrics-int.cert` | Geneva metrics client certificate |
+| `File` | `secrets/metrics-int.key` |  Geneva metrics client key |
+| `File` | `secrets/.dockerconfigjson` |  Docker config allowing to pull geneva images |
+| `File` | `test/manifests/normal/scaleup.yaml` |  Scale-up manifest (required for scale up/down tests) |
+| `File` | `test/manifests/normal/scaledown.yaml` |  Scale-down manifest (required for scale up/down tests) |
+
+#### Special considerations
+
+##### Scale up/down
+The scale up/down e2e test accepts two flags for customizing its functionality
+
+* `scaleUpManifest`: Path to the scale up manifest (default: `test/manifests/normal/scaleup.yaml`)
+* `scaleDownManifest`: Path to the scale down manifest (default: `test/manifests/normal/scaleup.yaml`)
+
+Partial updates are supported. These flags are required for any test focus which includes the scale up/down test.
+
+##### Key Rotation
+The key rotation up/down e2e test accepts two flags for customizing its functionality
+
+* `manifest`: Path to a customer manifest specifying a cluster for which keys should be rotated (default: `_data/manifest.yaml`)
+* `configBlob`: Path to an internal config blob for the cluster in question (default: `_data/containerservice.yaml`)
+
+These flags are required for any test focus which includes the key rotation test.
+
+##### Reentrant Updates
+The reentrant updates e2e test accepts one flag for customizing its functionality
+
+* `manifest`: Path to a customer manifest specifying a cluster for which reentrant updates should be tested (default: `_data/manifest.yaml`)
```

Comment:
This should not really be customizable and the same request that was used during cluster creation should be reused. Also the actual flag is `request` and not `manifest` since `manifest` is already used by the key rotation test.